dist/profile/manifests/kubernetes/resources/accountapp.pp

```diff
@@ -0,0 +1,134 @@
+#   Class: profile::kubernetes::resources::accountapp
+#
+#   This class deploy plugins jenkins website
+#
+#   Parameters:
+#     $image_tag:
+#       Set accountapp image tag.
+#     $election_candidates:
+#       coma separated list of candidates.
+#     $election_close:
+#       date election will close. yyyy/MM/dd
+#     $election_open:
+#       date app will start collecting votes. yyyy/MM/dd
+#     $election_logdir:
+#       collected votes directory.
+#     $jira_username:
+#       jira username
+#     $jira_url:
+#       jira url endpoint
+#     $jira_password:
+#       jira password 
+#     $ldap_manager_dn:
+#       ldap manager dn
+#     $ldap_new_user_base_dn:
+#       ldap new user base dn
+#     $ldap_password:
+#       ldap password
+#     $ldap_url:
+#       ldap endpoint
+#     $recaptcha_private_key:
+#       recaptcha private key 
+#     $recaptcha_public_key:
+#       recaptcha public key
+#     $smtp_server:
+#       smtp server
+#     $smtp_user:
+#       smtp user
+#     $smtp_password:
+#       smtp password
+#     $smtp_auth:
+#       smtp authentication (boolean)
+#     $url:
+#       account app url endpoint
+#
+#   Remark:
+#     `kubectl get service nginx --namespace nginx-ingress` return service public ip
+#     This public ip should be used for DNS record frontend_url -> this IP
+#     In order to create letsencrypt certificates
+#
+# Deploy accountapp resources on kubernetes cluster
+class profile::kubernetes::resources::accountapp (
+    String $election_close = '1970/01/02',
+    String $election_open = '1970/01/01',
+    String $election_logdir= '/var/log/accountapp/elections',
+    String $election_candidates = 'bob,alice',
+    String $image_tag = 'latest',
+    String $jira_username = 'jira_username',
+    String $jira_url = 'https://jira_url',
+    String $jira_password = 'jira_password',
+    String $ldap_manager_dn = 'cn=admin,dc=jenkins-ci,dc=org',
+    String $ldap_new_user_base_dn = 'ou=people,dc=jenkins-ci,dc=org',
+    String $ldap_password = 'ldap_password',
+    String $ldap_url = 'ldap://localhost:389/',
+    String $recaptcha_private_key = 'recaptcha_private_key',
+    String $recaptcha_public_key = 'recaptcha_public_key',
+    String $smtp_server = 'localhost',
+    String $smtp_user = '',
+    String $smtp_password = '',
+    Boolean $smtp_auth = true,
+    String $storage_account_name = '',
+    String $storage_account_key = '',
+    String $url = 'accounts.jenkins.io'
```

Comment:
Should the value for `$url` be a real URL, i.e. "https://accounts.jenkins.io"?

If it's supposed to be a domain name, please change the variable name to something like `domain_name`